src/DynamoCoreWpf/Views/Preview/PreviewControl.xaml

```diff
@@ -167,7 +167,52 @@
             <Border Background="White"
                     BorderThickness="1"
                     BorderBrush="{StaticResource BubblePreviewBorderColor}" />
-
```

Comment:
can we put a comment here on what this element is. (pin button) or something like that.